zerver/webhooks/facebook/fixtures/permissions_manage_pages.json

```diff
@@ -1,13 +1,22 @@
 {
-   "entry":[
```

Comment:
@nyan-salmon: Could you please explain in detail why you changed all of these fixtures? :)

Reply:
Sorry, I was not good.

I fixed it.

I changed some files name to conform to the facebook naming convention.